### PR TITLE
hardy2022re

### PR DESCRIPTION
**Pull request by Ian-Hardy**
## Project Name

hardy2022re

## Compiled PDF

[Compiled Pdf](2021/hardy2022re/journal/article.pdf)

## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `<paper_citekey>/journal/metadata.yaml`.

- [x ] `title` field exists and it starts with `[Re] `.
- [x ] `authors` contain author information with affiliations.
- [ x] [ORCID id](https://orcid.org/) is provided for authors.
- [x ] One contact author is added using `*`.
- [ x] Github `url` is provided in `code` section.
- [x ] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x ] `replication` section is populated with the necessary information about the paper you reproduced.
- [x ] `abstract` field is populated with your paper abstract (including double quotes)
- [x ] `domain` should contain "ML Reproducibility Challenge 2021"
- [x ] `review` section contains the url to the OpenReview forum of your paper.
- [ x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors

I encountered a couple of build errors that I haven't been able to resolve. They are:
- For some reason the generated PDF contains two titles sections, I'm not sure why
- My bibliography is not being generated. I get some errors about undefined citations (see log.) I have confirmed that my openreview document (the reorganized Latex) renders correctly in Overleaf. 
- The title(s) are still anonymized 

Here is the link to my build log: https://pastebin.com/UxsFyX5p

### CODE REVIEW

File: 2021/hardy2022re/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,19 @@
+% Import packages here
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage{hyperref}
+
+\usepackage[]{neurips_2019}
```
Comment (koustuvsinha) [inline 2021/hardy2022re/openreview/packages.tex:5]: Please comment this import (`neurips2019`) as we are using ReScience styles.

File: 2021/hardy2022re/openreview/packages.tex [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+% to avoid loading the natbib package, add option nonatbib:
+%     \usepackage[nonatbib]{neurips_2019}
+
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+
+\usepackage[dvipsnames]{xcolor}
```
Comment (koustuvsinha) [inline 2021/hardy2022re/openreview/packages.tex:18]: Please comment this import (`xcolor`) as ReScience style files does not work with this (check Readme).

File: 2021/hardy2022re/openreview/content.tex [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+As mentioned in my summary, I did not have contact with the authors throughout this process. It was only upon drafting my report that I learned it was encouraged to contact the original authors; in the future, I think it would be a great idea to communicate with them sooner. I reached out with a preprint of the report for any feedback or suggestions, but did not hear back. 
+
+% <None so far, pending reaching out?>
+
+% Original text:
+% "Document the extent of (or lack of) communication with the original authors. To make sure the reproducibility report is a fair assessment of the original research we recommend getting in touch with the original authors. You can ask authors specific questions, or if you don't have any questions you can send them the full report to get their feedback before it gets published." 
+
+\bibliographystyle{unsrt}
+\bibliography{ref}
```
Comment (koustuvsinha) [inline 2021/hardy2022re/openreview/content.tex:173]: Please comment out both of these lines (172-173) as ReScience template handles bibliography placement with `/printbibliography` call in `journal/article.tex`.

**Comment by koustuvsinha** [comment]
Hi @Ian-Hardy , I have added some feedback, once you comment these lines your build should work as expected.

**Comment by Ian-Hardy** [comment]
Thank you so much for these notes! It now appears to compile successfully.

**Pull request merged**